sources/php5/KalturaClientBase.php

```diff
@@ -823,7 +823,18 @@ public function jsObjectToClientObject($value)
 			
 			if(!isset($value->objectType))
 			{
-				throw new KalturaClientException("Response format not supported - objectType is required for all objects", KalturaClientException::ERROR_FORMAT_NOT_SUPPORTED);
+				if (isset($value->result))
```

Comment:
This is a very bad idea, it will crash on any OVP object that happens to have a property called "result".
Before checking it you should check if "objectType" property is missing, that's the indication it's OTT, only then check if it's result or error.

Reply:
@tan-tan-kanarek this check will only happen if !isset($value->objectType) which is exactly what you wrote.